src/BTrees/_datatypes.py

```diff
@@ -20,6 +20,13 @@
 from struct import Struct
 from struct import error as struct_error
 
+try:
+    from abc import ABC
+except ImportError:
+    # Python < 3.4
```

Comment:
Maybe mention Python 2.7 here as it is the only version < 3.4 which is currently supported.
This could ease dropping Python 2 support later on when it is appropriate.

Reply:
No problem, thanks for the suggestion.